File: src/main/java/com/DavidM1A2/AfraidOfTheDark/common/block/gravewood/BlockGravewood.java
/*
 * Author: David Slovikosky
 * Mod: Afraid of the Dark
 * Ideas and Textures: Michael Albertson
 */
package com.DavidM1A2.AfraidOfTheDark.common.block.gravewood;

import com.DavidM1A2.AfraidOfTheDark.common.block.core.AOTDLog;

import net.minecraft.block.Block;

public class BlockGravewood extends AOTDLog
{
	/*
	 * Define a gravewood block
	 */
	public BlockGravewood()
	{
		super();
		this.setStepSound(Block.soundTypeWood);
		this.setUnlocalizedName("gravewood");
	}
}

File: src/main/kotlin/com/davidm1a2/afraidofthedark/common/block/gravewood/BlockGravewood.kt
package com.davidm1a2.afraidofthedark.common.block.gravewood

import com.davidm1a2.afraidofthedark.common.block.core.AOTDBlockLog
import net.minecraft.block.material.Material
import net.minecraft.block.material.MaterialColor

/**
 * Class representing a gravewood log block
 *
 * @constructor just sets the registry and unlocalized name
 */
class BlockGravewood : AOTDBlockLog("gravewood", MaterialColor.WOOD, Properties.create(Material.WOOD, MaterialColor.WOOD))
